src/services/product.service.js

```diff
@@ -1,7 +1,13 @@
-import { Products } from '../database/models/index';
+import { Products, Categories } from '../database/models/index';
 
 export class ProductService {
   static async createProduct(product) {
     return await Products.create(product);
   }
+  static async searchProduct(searchQuery){
+    return await Products.findAll({
+      where: searchQuery,
+      include: [{ model: Categories }]
+    });
```

Comment:
use sequelize like Operation not where.  [Model querying](https://sequelize.org/docs/v6/core-concepts/model-querying-finders/)

Reply:
Here I have used `like operator` in middleware where I formatted that object `searchQuery`